lametro/events.py

```diff
@@ -1,10 +1,21 @@
 import datetime
+import logging
 
 from legistar.events import LegistarAPIEventScraper
 from pupa.scrape import Event, Scraper
 from legistar.base import LegistarScraper
 
 
+LOGGER = logging.getLogger(__name__)
+
+class UnmatchedEventError(Exception):
```

Comment:
the base exception class returns whatever argument is passed to it as its message. in order to return your custom message, you can either `super().__init__(message)` at the end of your custom `__init__` method, or define the `__str__` dunder method on your custom exception. i prefer the first approach – what do you think?